vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -93,6 +93,9 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 		if errno == syscall.ECONNRESET {
 			msg += " Check that ESX service is running."
+			msg += " Check vmware tools or open vm tools is installed."
```

Comment:
We need to document somewhere how to do all of the above! and possibly include link here.